tests/codegen/sanitizer/cfi/external_weak_symbols.rs

```diff
@@ -0,0 +1,23 @@
+// Verifies that type metadata identifiers for trait objects are emitted correctly.
```

Comment:
[...] for weakly-linked symbols are emitted correctly?

Reply:
Yes indeed! Thanks Ramon :)